prep-base.sh

```
@@ -41,12 +41,13 @@ nix develop -i ${keep[@]} -c bash \
   -c '(cd lib/rain.interpreter/lib/rain.interpreter.interface/lib/rain.math.float && rainix-rs-prelude)'
 
 echo "Setting up rain.tofu.erc20-decimals..."
-(cd lib/rain.interpreter/lib/rain.tofu.erc20-decimals && nix develop -c forge build)
+nix develop -i ${keep[@]} -c bash \
```

Shellcheck (0.11.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)